CMakeLists.txt

```diff
@@ -32,8 +32,15 @@ endif()
 
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE CACHE BOOL "Add paths to linker search and installed rpath")
 
-# Get dependencies
-include(cmake/Dependencies.cmake)
+# rocm-cmake contains common cmake code for rocm projects to help
+# setup and install
+find_package(ROCM 0.6 QUIET CONFIG PATHS /opt/rocm)
```

Comment:
This check should probably be REQUIRED instead of QUIET, so that it causes a clear failure if it can't find rocm-cmake.

Reply:
Otherwise if it fails to find rocm-cmake it will just fail when it tries to call a function from rocm-cmake (like `rocm_create_package` later in this file).